src/moonlink/src/storage/mooncake_table/snapshot.rs

```diff
@@ -464,7 +464,15 @@ impl SnapshotTableState {
             .await;
         evicted_data_files_to_delete.extend(compaction_evicted_files);
 
+        // Remap uncommitted deletion logs.
+        self.remap_uncommitted_deletion_logs_after_compaction(&mut task);
```

Comment:
one TODO is to combine two remap operations, after https://github.com/Mooncake-Labs/moonlink/pull/1045